Velaptor/Reactables/ShutDownReactable.cs

```diff
@@ -2,29 +2,14 @@
 // Copyright (c) KinsonDigital. All rights reserved.
 // </copyright>
 
-using Velaptor.Reactables.Core;
-using Velaptor.Reactables.ReactableData;
-
 namespace Velaptor.Reactables;
 
+using Core;
+using ReactableData;
+
 /// <summary>
 /// Creates a reactable to send push notifications to signal that the application is shutting down.
```

Comment:
Refactor - I've noticed that some docs say push notifications (plural), others say a push notification (singular).  Not sure if some instances warrant plural vs. singular.  If not, you'll need to refactor all of them to either singular or plural. 

```suggestion
/// Creates a reactable to send push notifications to signal that the application is shutting down.
```